[2024-11-10T01:58:52.925+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T01:58:49.541844+00:00/task_id=natural_gas_spot_prices_extraction permission to 509
[2024-11-10T01:58:52.926+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T01:58:49.541844+00:00/task_id=natural_gas_spot_prices_extraction permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T01:58:49.541844+00:00/task_id=natural_gas_spot_prices_extraction'
[2024-11-10T01:58:52.976+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T01:58:49.541844+00:00/task_id=natural_gas_spot_prices_extraction permission to 509
[2024-11-10T01:58:52.977+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T01:58:49.541844+00:00/task_id=natural_gas_spot_prices_extraction permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T01:58:49.541844+00:00/task_id=natural_gas_spot_prices_extraction'
[2024-11-10T01:58:53.054+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: natural_gas_spot_prices_etl_pipeline.natural_gas_spot_prices_extraction manual__2024-11-10T01:58:49.541844+00:00 [queued]>
[2024-11-10T01:58:53.069+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: natural_gas_spot_prices_etl_pipeline.natural_gas_spot_prices_extraction manual__2024-11-10T01:58:49.541844+00:00 [queued]>
[2024-11-10T01:58:53.070+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-11-10T01:58:53.095+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): natural_gas_spot_prices_extraction> on 2024-11-10 01:58:49.541844+00:00
[2024-11-10T01:58:53.106+0000] {standard_task_runner.py:57} INFO - Started process 19612 to run task
[2024-11-10T01:58:53.112+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'natural_gas_spot_prices_etl_pipeline', 'natural_gas_spot_prices_extraction', 'manual__2024-11-10T01:58:49.541844+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/eia_natural_gas_spot_prices_etl_pipeline_dag/eia_natural_gas_spot_prices_etl_pipeline_dag.py', '--cfg-path', '/tmp/tmp3dwuymsd']
[2024-11-10T01:58:53.116+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask natural_gas_spot_prices_extraction
[2024-11-10T01:58:53.207+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T01:58:49.541844+00:00/task_id=natural_gas_spot_prices_extraction permission to 509
[2024-11-10T01:58:53.208+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T01:58:49.541844+00:00/task_id=natural_gas_spot_prices_extraction permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=natural_gas_spot_prices_etl_pipeline/run_id=manual__2024-11-10T01:58:49.541844+00:00/task_id=natural_gas_spot_prices_extraction'
[2024-11-10T01:58:53.212+0000] {task_command.py:415} INFO - Running <TaskInstance: natural_gas_spot_prices_etl_pipeline.natural_gas_spot_prices_extraction manual__2024-11-10T01:58:49.541844+00:00 [running]> on host 16801ac1f6e4
[2024-11-10T01:58:53.396+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='natural_gas_spot_prices_etl_pipeline' AIRFLOW_CTX_TASK_ID='natural_gas_spot_prices_extraction' AIRFLOW_CTX_EXECUTION_DATE='2024-11-10T01:58:49.541844+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-10T01:58:49.541844+00:00'
[2024-11-10T01:58:55.648+0000] {logging_mixin.py:151} INFO - Error retrieving metadata: An error occurred (NoSuchKey) when calling the GetObject operation: The specified key does not exist.
[2024-11-10T01:58:55.651+0000] {logging_mixin.py:151} INFO - {'X-Params': '{"api_key": "qLjNQdfhg4Gs12Ur7HnXm83dZwMNhmCmxvX5gs4f", "frequency": "daily", "data": ["value"], "facets": {"series": ["RNGWHHD"]}, "sort": [{"column": "period", "direction": "asc"}], "length": 5000, "start_date": "1999-01-04", "offset": 0}', 'Content-Type': 'application/json'}
[2024-11-10T01:58:58.901+0000] {logging_mixin.py:151} INFO - Data is:  []
[2024-11-10T01:58:58.904+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-11-10T01:58:58.923+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=natural_gas_spot_prices_etl_pipeline, task_id=natural_gas_spot_prices_extraction, execution_date=20241110T015849, start_date=20241110T015853, end_date=20241110T015858
[2024-11-10T01:58:58.965+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-10T01:58:59.015+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
